src/keycloak/Dockerfile

```diff
@@ -9,7 +9,8 @@
 #     IBM Corporation - initial API and implementation
 ################################################################################
 
-FROM jboss/keycloak:8.0.0
+FROM jboss/keycloak:latest
```

Comment:
I don't think we always want to build with latest keycloak, we had problems in the past where we picked up some unwanted breaking changes.

Is there a newer version of keycloak that you require?

Also, should this change be in a `ppc64le` version of the dockerfile?

You may want to discuss this part with @markcor11

Reply:
The script picks up Dockerfiles for pfe and performance specific to architecture. For gatekeeper and keycloak, I dont find any such logic, hence changed the same Docker file.

Reply:
Changed tag for keycloak image.